src/jgikbase/test/idmapping/test_utils.py

```diff
@@ -1,9 +1,72 @@
-"""
+import os
+import socket
+from contextlib import closing
+from pathlib import Path
+import configparser
 
-@author: gaprice@lbl.gov
-"""
+MONGO_EXE = 'test.mongo.exe'
+TEST_TEMP_DIR = 'test.temp.dir'
+MONGO_USE_WIRED_TIGER = 'test.mongo.wired_tiger'
+KEEP_TEMP_DIR = 'test.temp.dir.keep'
+
+TEST_CONFIG_FILE_SECTION = 'idmappingservicetest'
+
+TEST_FILE_LOC_ENV_KEY = 'IDMAP_TEST_FILE'
+
+_CONFIG = None
+
+
+def get_mongo_exe() -> Path:
+    return Path(os.path.abspath(_get_test_property(MONGO_EXE)))
+
+
+def get_temp_dir() -> Path:
+    return Path(os.path.abspath(_get_test_property(TEST_TEMP_DIR)))
+
+
+def get_use_wired_tiger() -> bool:
```

Comment:
One idea is instead of a bunch of getter functions, you could define a dict of config values:
```py
test_config = {
  'mongo_exe': _get_test_property('test.mongo.exe'),
  'temp_dir': _get_test_property('test.temp.dir'),
   ...etc
}
```

Reply:
I prefer the functions since an editor can introspect the functions but not the dict. Otherwise I don't see that it makes a huge difference.